CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Changelog
 
+## 1.10.0
+
+- ### Features
+
+  - **`wrangler dev` now requires that you specify "preview" versions of KV namespaces - [EverlastingBugstopper], [issue/1032] [pull/1357] [pull/1360]**
+
+    We're preparing to make `wrangler dev` run entirely on Cloudflare's edge, giving it access to functionality that's currently missing in the preview environment, and more accurately reproducing what happens when your Worker is deployed to production. This means that, among other things, your Workers can access the Cache API and your production KV Namespaces.
```

Comment:
i dont think this block is necessary. this affects users who arent running on the edge as well, as you can access prod namespaces from our existing preview environment

Reply:
Updated, thanks!